image_tools/CMakeLists.txt

```diff
@@ -11,40 +11,50 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+set(ADD_NODE_PATH ../../rclcpp/rclcpp_components/cmake/rclcpp_components_add_node.cmake)
+
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
 find_package(sensor_msgs REQUIRED)
 find_package(std_msgs REQUIRED)
-
+find_package(rcutils REQUIRED)
 find_package(OpenCV REQUIRED)
 
 include_directories(include)
+include(${ADD_NODE_PATH})
 
-add_executable(cam2image
-  src/burger.cpp
+rclcpp_components_add_node(TARGET showimage_component SRCS
```

Comment:
With ros2/rclcpp#758, we can rename our nodes back to their original names (e.g. `showimage` instead of `showimage_component`).